src/staging/src/components/advisories/advisoryCard.js

```diff
@@ -0,0 +1,258 @@
+  },
+  dateBanner: {
+    // only show for mobile
+    [theme.breakpoints.up('sm')]: {
+      display: "none" // don't show for mobile, see dateBanner
+    },
+    borderRadius: "4px 4px 0px 0px",
+    padding: "3px 8px",
+    color:"#fff"
+  },
+  dateArea: {
+    [theme.breakpoints.down('sm')]: {
+      display: "none" // don't show for mobile, see dateBanner
+    },
+    [theme.breakpoints.up('sm')]: {
+      padding: ".5rem 1.75rem 0 0.5rem",
+      display: "inline-flex",
+    },
+  },
+  dateCircle: { // Not a circle for mobile (xs)
+    background: '#656565', // default bg, e.g. "grey alert"
+    color: '#fff',
+    [theme.breakpoints.down('sm')]: {
+      padding: "3px 8px",
+      borderRadius: "4px"
+    },
+    [theme.breakpoints.up('sm')]: {
+      width: '7.5rem',
+      height: '7.5rem',
+      paddingTop: '1.15rem',
+      borderRadius: '50%',
+      textAlign: 'center',      
+    },
+  },
+  dateStr: {
+    [theme.breakpoints.down('sm')]: {
+      display: "block"
+    },
+    [theme.breakpoints.up('sm')]: {
+      display: "none"
+    }
+  },
+  dateDate: {
+    [theme.breakpoints.down('sm')]: {
+      display: "none"
+    },
+    [theme.breakpoints.up('sm')]: {
+      fontSize: "2.5rem",
+      lineHeight: "3.2rem"
+    }
+  },
+  dateMonth: {
+    [theme.breakpoints.down('sm')]: {
+        display: "none"
+    },
+    [theme.breakpoints.up('sm')]: {
+      fontSize: "1.25rem",
+      lineHeight: "1.05rem"
+    }
+  },
+  dateUnknown: {
+    [theme.breakpoints.down('sm')]: {
+        display: "inline-block"
+    },
+    [theme.breakpoints.up('sm')]: {
+      fontSize: "1.25rem",
+      lineHeight: "5rem"
+    }
+    },
+  blueAlert: {
+    background: "#2464a4",
+  },
+  redAlert: {
+    background: "#d8292f",
+  },
+  yellowAlert: {
+    background: "#fcba19",
+  },
+  contentMargin: {
+    width: "10rem",
+    display: "inline-flex"
+  },
+  contentArea: {
+    [theme.breakpoints.up('sm')]: {
+      display: "flex",
+      flexDirection: "column",
+      justifyContent: "center"
+    }
+  },
+  detailsArea: {
+    [theme.breakpoints.up('sm')]: {
+      marginLeft: "10.5rem",
+    } 
+  },
+  noDetails: { // For cards withouth details, no expand icon and regular cursor
+    '& .MuiAccordionSummary-expandIcon': {
+      display:"none"
+    },
+    '& .MuiAccordionSummary-root.Mui-expanded': {
+      margin:0
+    },
+    '& .MuiButtonBase-root': {
+      cursor:"default !important"
+    }
+  },
+  yearHeader: { // For the first card for each ear
+    fontSize: "2rem",
+    marginTop: "1rem"
+  }
+}));
+    
+const AdvisoryCard = ({ advisory, index }) => {
+
+    const classes = useStyles();
+
+    return (
+        <>
+          <Grid item xs={12}
+            className={classes[advisory.detailsClass]}
+            key={advisory.id} thing={advisory.isFirstInYear.toString()}>
+            <div className={((advisory.isFirstInYear || (index === 0)) && (!advisory.advisoryDateObj.dateUnknown)) ? classes.yearHeader : "hidden"}>
```

Comment:
I would make this a conditional render instead of using `display: none;` I think if you do hidden, react has to go to the work of putting it into the DOM just to not show it.